src/Expression/SymbolicaRuntimeException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Symbolica.Expression
+{
+    [Serializable]
+    public class SymbolicaRuntimeException : SymbolicaException
```

Comment:
Is this one supposed to be used for actual exceptions? I'm not sure the name is too obvious here as all exceptions are runtime exceptions aren't they?

Reply:
As discussed, it's gone.